### ISSUE TITLE
Configure/automate metrics start dates 

### ISSUE THREAD

**Issue by helbashandy** [NCEAS/metacatui#1537]
**Describe the feature you'd like**
The dates for downloads and citations graphs starts on July 2012, the inception of DataOne. On our repository that shows as a large portion of the plots as empty so it would be ideal if the plots starts on the data of which the first data point starts. Or if that doesn't make sense for other member nodes, maybe we could configure the start dates for those metrics?

**Comment by mbjones** [comment]
Agreed. We have a bunch of repositories that have metrics starting both earlier and later than 2012, and so @rushirajnenuji is working on getting the plots to scale to the data that are available. Please comment @rushirajnenuji if you think this is tractable in your current work.

**Comment by laurenwalker** [comment]
Hi @helbashandy, can you clarify which version of MetacatUI has this behavior? I believe the newest version, which uses the new slider bar charts, does not have this issue. But the older bar charts in older versions of MetacatUI had a hard-coded start date to start the charts from. I just peeked at https://data.ess-dive.lbl.gov/profile and see that you might have figured out how to change that start date already because it starts in 2018? Thanks

**Comment by helbashandy** [comment]
@laurenwalker Our production currently is on an old metacatUI version (2.8.5). We're currently in the process of upgrading to metacatUI 2.12.0 and since our release candidate is still on development system the metrics doesn't load so we don't know how it would look like on production. But my prediction is that the start dates would be the same as the ones on the ADC (https://arcticdata.io/catalog/profile)

**Comment by laurenwalker** [comment]
Hi @helbashandy it looks like your latest UI update shows the downloads and view charts starting in  2017 now. Is this issue resolved or did you have to override something in your theme to get that start date? If so, we could work that override into the AppConfig.

**Comment by helbashandy** [comment]
Hi @laurenwalker - After our deployment last week we were able to see how it looked like and it started from May 2017, we didn't override the start date or add a config for it, not sure why it started from May 2017 on our end. It would still be great if there's a config for it though since even with the May 2017 date, our public data started on March 2018.
